Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+public class PlayerController : MonoBehaviour {
+
+    [SerializeField]
+    private float playerAcceleration, maxSpeed, jumpForceForward, jumpForceUp, bulletSpeed, recoilSpeed, overheatAdd, overheatSubtract;
```

Comment:
Add some ToolTip attributes for these to clarify them. https://docs.unity3d.com/ScriptReference/TooltipAttribute.html